x-pack/plugins/ml/public/application/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -145,27 +175,25 @@ export const AnalyticsPanel: FC<Props> = ({
                 </h3>
               </EuiText>
             </EuiFlexItem>
-            {analyticsStats !== undefined && (
-              <EuiFlexItem grow={false} className="mlOverviewPanel__statsBar">
-                <StatsBar stats={analyticsStats} dataTestSub={'mlOverviewAnalyticsStatsBar'} />
-              </EuiFlexItem>
-            )}
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup gutterSize={'s'} alignItems="center">
+                {analyticsStats !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <StatsBar stats={analyticsStats} dataTestSub={'mlOverviewAnalyticsStatsBar'} />
+                  </EuiFlexItem>
+                ) : null}
+                <EuiFlexItem grow={false}>
+                  <EuiButton size="m" fill href={manageJobsLink}>
+                    {i18n.translate('xpack.ml.overview.analyticsList.manageJobsButtonText', {
+                      defaultMessage: 'Manage jobs',
+                    })}
+                  </EuiButton>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
           </EuiFlexGroup>
           <EuiSpacer />
           <AnalyticsTable items={analytics} />
```

Comment:
Were you planning on adding 'Running time' to the columns in this table? Wonder if there's room for another column?

Reply:
At the moment it's not possible, I've open an issue for ES repo https://github.com/elastic/elasticsearch/issues/81814